src/sentry/utils/snuba.py

```diff
@@ -729,6 +760,26 @@ def snuba_query(params):
     return results
 
 
+def bulk_snql_raw_query(params, headers, referrer, entity):
+    legacy_body, forward, reverse, thread_hub = params
+    with thread_hub.start_span(op="snuba.snql", description="snql-sdk") as span:
+        span.set_tag("referrer", referrer)
+        snql_query = json_to_snql(legacy_body, entity)
```

Comment:
Yes I am going to add a function that takes the SnQL query directly, instead of converting a legacy query. That will happen in a subsequent PR. Ideally we would migrate all of Sentry to use the new function, but I know that will take a while so I thought I would start here.

Reply:
👍 to having a function that handles SnQL more directly. Once you have this feature flag bumped up, would this code path migrate to the new functions?

Reply:
Over time definitely. I wanted this function here so that we could use the SnQL API with minimal work from the product teams. However the new functions can be used as much/as little as the product teams want.